project.py

```diff
@@ -153,3 +159,12 @@ def findroot():
             path = os.path.realpath(os.path.join(path, ".."))
     stderr.write("There doesn't seem to be a KnightOS project here. Did you run `knightos init`?\n")
     exit(1)
+
+#Currently there's no way to overwrite a pre-existing symlink
+def force_symlink(file1, file2):
+    try:
+        os.symlink(file1, file2)
+    except OSError as e:
```

Comment:
Couldn't you just start off by deleting file2? Will it throw an error if you try to delete a non-existant file?